modules/nlu/src/views/full/intents/utterances-state-utils.ts

```diff
@@ -1,7 +1,8 @@
 import _ from 'lodash'
 import { MarkJSON, NodeJSON, TextJSON, Value, ValueJSON } from 'slate'
 
-import { ParsedSlot, parseUtterance } from '../../../backend/utterance/utterance-parser'
+// this mess i temporary...
+import { ParsedSlot, parseUtterance } from '../../../../../../src/bp/nlu-core/utterance/utterance-parser'
```

Comment:
is this ok?

Reply:
@allardy suggested to add this in common making this much easier ... but since we're getting rid of the NLU then i guess this is ok

Reply:
the UI gets removed so it's ok